app/views/ctc/ctc_pages/home.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <%= render 'shared/facebook_pixel' %>
 <main role="main" class="ctc-home">
-  <%= render 'ctc_offseason_warning' %>
```

Comment:
i dont know whether to completely remove this, or leave it and comment it out so that its easier to toggle if we end up wanting to use it again during/after the season?

Reply:
I'm inclined to say remove it fully. It'll be easy to bring back. Not a blocker.

Reply:
This reminds me that I'm not sure we have written down the open/close procedures for GetCTC and what the expectations are on different dates. Do you want to talk with Yvonne/Kelly about that? Want me to?